application/config.py

```diff
@@ -6,28 +6,26 @@ class Config(object):
     TESTING = False
     CSRF_ENABLED = False
     # SECRET = 'this-really-needs-to-be-changed'
-    SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
+    SQLALCHEMY_DATABASE_URI = "postgresql://localhost/auctionable_change_api"
     # SQLALCHEMY_TRACK_MODIFICATIONS = False
```

Comment:
Let's see if we can get this back into the .env file

Reply:
Ahhh... good call. I think I can make that modification & push up before we merge this.

Reply:
Pushed up additional change to do exactly as Stephanie suggested.